File: ios/stereoCamera/stereoCamera/Comm/Commands/SendConnectionPause.swift
//
//  SendConnectionPause.swift
//  stereoCamera
//
//  Created by hallmarklabs on 7/9/18.
//  Copyright © 2018 cody. All rights reserved.
//

import Foundation

class SendConnectionPause : Command
{
    override init()
    {
        super.init()
        cmdtype = CommandTypes.RECEIVE_CONNECTION_PAUSE
        expectsResponse = false
    }
}

File: android/app/src/main/java/com/munger/stereocamera/ip/command/commands/SendConnectionPause.java
package com.munger.stereocamera.ip.command.commands;

import com.munger.stereocamera.ip.command.Command;

public class SendConnectionPause extends Command
{
	public SendConnectionPause()
	{
		super();
		cmdtype = Type.RECEIVE_CONNECTION_PAUSE;
		expectsResponse = false;
	}
}
